new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItCoherenceTests.java

```diff
@@ -0,0 +1,395 @@
+  private static final String COHERENCE_MODEL_PROP = "coherence-wdt-config.properties";
+  private static final String COHERENCE_IMAGE_NAME = "coherence-image";
+
+  private static String domainUid = "coh-domain1";
+  private static String clusterName = "cluster-1";
+  private static String adminServerPodName = domainUid + "-admin-server";
+  private static String managedServerPrefix = domainUid + "-managed-server";
+  private static String containerName = "weblogic-server";
+  private static int replicaCount = 2;
+
+  private static String opNamespace = null;
+  private static String domainNamespace = null;
+  private static ConditionFactory withStandardRetryPolicy = null;
+  private static Map<String, Object> secretNameMap;
+  private static LoggingFacade logger = null;
+
+  /**
+   * Install Operator.
+   *
+   * @param namespaces list of namespaces created by the IntegrationTestWatcher by the
+   *                   JUnit engine parameter resolution mechanism
+   */
+  @BeforeAll
+  public static void init(@Namespaces(2) List<String> namespaces) {
+    logger = getLogger();
+    // create standard, reusable retry/backoff policy
+    withStandardRetryPolicy = with().pollDelay(2, SECONDS)
+      .and().with().pollInterval(10, SECONDS)
+      .atMost(5, MINUTES).await();
+
+    // get a new unique opNamespace
+    logger.info("Assigning a unique namespace for Operator");
+    assertNotNull(namespaces.get(0), "Namespace list is null");
+    opNamespace = namespaces.get(0);
+
+    // get a new unique domainNamespace
+    logger.info("Assigning a unique namespace for Domain");
+    assertNotNull(namespaces.get(1), "Namespace list is null");
+    domainNamespace = namespaces.get(1);
+
+    // install and verify operator
+    installAndVerifyOperator(opNamespace, domainNamespace);
+  }
+
+  // This method is needed in this test class, since the cleanup util
+  // won't cleanup the images.
+  @AfterAll
+  void tearDown() {
+    // Delete domain custom resource
+    logger.info("Delete domain custom resource in namespace {0}", domainNamespace);
+    assertDoesNotThrow(() -> deleteDomainCustomResource(domainUid, domainNamespace),
+        "deleteDomainCustomResource failed with ApiException");
+    logger.info("Deleted Domain Custom Resource " + domainUid + " from " + domainNamespace);
+  }
+
+  /**
+   * Create a WebLogic domain with a Coherence cluster and deploying it using WDT
+   * Test rolling restart of Coherence managed servers and verify
+   * that data are not lost during a domain restart.
+   */
+  @Test
+  @DisplayName("Create domain with a Coherence cluster using WDT and test rolling restart")
+  @Slow
+  @MustNotRunInParallel
+  public void testCohernceServerRollingRestart() {
+    final String successMarker = "CACHE-SUCCESS";
+
+    // create and verify WebLogic domain image using model in image with model files
+    String miiImage = createAndVerifyDomainImage();
```

Comment:
change the variable name to domImage to reflect the DomainHomeInImage configuration

Reply:
changed